docs/source/ngv.rst

```diff
@@ -15,6 +15,128 @@ The NGV simulation requires three node populations, each stored in separate file
 The diagram also illustrates the various connections among the populations, as described by the edge files (one per connection type):
 
 - **Neuron <-> Neuron connections** are stored in the `neocortex_neurons__chemical_synapse` file. These are simple synaptic connections (type: chemical).
```

Comment:
We write "neocortex", but I suppose that is because we haven't built NGV components for other brain regions? In the future we might need to rephrase these connection types to simply be "neuronal"

Reply:
I follow the current convention. I prefer docs that state what is going on now than flexible docs that may be relevant longer in the future by avoiding to use hardcoded names